crmsh/utils.py

```diff
@@ -2649,4 +2649,46 @@ def ping_node(node):
     rc, _, err = get_stdout_stderr("ping -c 1 {}".format(node))
     if rc != 0:
         raise ValueError("host \"{}\" is unreachable: {}".format(node, err))
+
+
+def is_quorate(expected_votes, actual_votes):
```

Comment:
I wonder if it wouldn't be better to calculate the `expected_votes` and `actual_votes` in this method directly.
Is there any reason you decided them to pass in the parameters?

Reply:
@arbulu89 This methond is called by `bootstrap.evaluate_qdevice_quorum_effect`
**Before** adding qdevice on a running cluster or removing qdevice from a cluster, I should evaluate whether the adding/removing action will cause quorum lost
So I need to create a new method to calculate that result giving the current expected_votes and actual_votes from `corosync-quorumtool -s`